Package.swift

```diff
@@ -45,8 +45,6 @@ let package = Package(
             path: "GTMAppAuth/Sources",
             sources: [
```

Comment:
We can remove this `source` section.

Reply:
Yeah, I'm not sure the [sources property](https://developer.apple.com/documentation/swift_packages/target/2880339-sources) was needed before either.